app/components/PillLabelBox.tsx

```diff
@@ -0,0 +1,62 @@
+import { tw } from '@helpwave/common/twind/index'
+
+export type PillLabelBoxProps = {
+  unscheduledTasks: number,
+  inProgressTasks: number,
+  doneTasks: number
+}
+
+const PillLabelBox = ({ unscheduledTasks, inProgressTasks, doneTasks }: PillLabelBoxProps) => {
+  const between = '0.2rem'
+  const height = '0.75rem' // equivalent to h-6 of tailwind
+  return (
+    <div className={tw('flex flex-row h-6')}>
+      <div className={tw('flex flex-row bg-hw-label-1-background rounded-l-md pl-2 pr-1 items-center text-hw-label-1-text')}>
+        <div className={tw(`rounded-full w-2 h-2 bg-hw-label-1-accent mr-1`)}/>
+        {unscheduledTasks.toString()}
+      </div>
+      <div
+        className={tw('w-0 h-0 border-hw-label-1-background border-solid border-r-transparent border-b-transparent')}
+        style={{
```

Comment:
[You can use arbitrary values in tailwind.](https://tailwindcss.com/docs/border-width#arbitrary-values) We can adopt this here like `className={tw('... border-y-[0.75rem] border-x-[0.2rem]')}` instead of the inline styles.

Reply:
Good to know, thank you!